build-logic/jvm/src/main/kotlin/build-logic.java.gradle.kts

```diff
@@ -0,0 +1,36 @@
+import com.github.vlsi.gradle.dsl.configureEach
+
+plugins {
+    id("java")
+    id("com.github.vlsi.crlf")
+    id("com.github.vlsi.gradle-extensions")
+    id("build-logic.test-base")
+    id("build-logic.build-params")
+    id("build-logic.test-junit5")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
switch to jvmToolChain and make sure we can still overwrite this via JAVA_VERSION env (we use it e.g. in build-ubuntu.yaml) -- see current build.gradle.kts in main

Reply:
Well, the toolchains were not there when I created the PR

Reply:
I know, just wanted to be sure we don't drop it again by accident